### DISCUSSION TITLE
New files in CG

### DISCUSSION THREAD

**Discussion by moonso** [Clinical-Genomics/cg#1019] [General]
Hi, I've noticed that there are some new files in the root of the CG repo. These are https://github.com/Clinical-Genomics/cg/blob/master/housekeeper, https://github.com/Clinical-Genomics/cg/blob/master/status and https://github.com/Clinical-Genomics/cg/blob/master/status_db. Looks like they where added by @Mropat in the refactor analysisAPI pr. 

I guess these has to do with some kind of test environment. Could we remove them or place them in a folder?

**Comment by Mropat** [comment]
Just remove them. Not sure how they got created but probably via some pycharm search/refactor shenanigans
